kover-gradle-plugin/examples/android/dynamic/dyn/build.gradle.kts

```diff
@@ -0,0 +1,82 @@
+plugins {
+    id ("com.android.library")
```

Comment:
Shouldn't this be a dynamic feature module in this example?

Reply:
Yes, thanks!
My bad)

Reply:
Yes, looks like plugin should be `com.android.dynamic-feature`